src/Components/TicTacToe/Square.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export default function Square({value, onClick}) {
+  const extendedStyle = {
+    ...style,
+    color: value === "X" ? "red" : "blue",
```

Comment:
When doing the styles separately, you could create different classes for empty, cross, and circle, and assigning these to the square, which makes it easy to give each of them their own style